src/main/java/ch/ibw/clientServer/server/dateReply/DateServer.java

```diff
@@ -15,12 +16,52 @@
  * @see <https://cs.lmu.edu/~ray/notes/javanetexamples/>
  */
 public class DateServer {
+    private static SimpleDateFormat time = new SimpleDateFormat("'Es ist gerade 'H'.'mm' Uhr.'");
+    private static SimpleDateFormat date = new SimpleDateFormat("'Heute ist 'EEEE', der 'dd.MM.yy");
+
     public static void main(String[] args) throws IOException {
+
+
+
         try (ServerSocket listener = new ServerSocket(6060)) {
             System.out.println("DateServer läuft");
+
+            boolean serverAN = true;
+
+
             try (Socket socket = listener.accept()) {   // Warte auf Clientverbindung
-                PrintWriter out = new PrintWriter(socket.getOutputStream(), true);
-                out.println(new Date().toString());     // Sende Antwort an Client
+                System.out.println("[Server] verbunden.");
+                while (serverAN)  {
```

Comment:
Hi Claudia
Das ist eine gute Lösung. Damit kann der Client in derselben Verbindung mehrere Anfragen schicken, oder?

Sobald dieser Client allerdings die Verbidung abbricht, beendet auch der Server. Was musst du tun, damit der Server nicht beendet, sondern auf die nächste Verbindung wartet?